README.md

```diff
@@ -21,8 +21,12 @@ Our script does the following:
 - Copies used images to an output folder, to allow them to be moved to a new home.
 - Afterwards, it asks if you want to try downloading the original size images using [download_better_images.py](https://raw.githubusercontent.com/timhutton/twitter-archive-parser/main/download_better_images.py).
 
+You can then convert the md files to a single HTML page with:
+
+    python3 convert_to_html.py
```

Comment:
Given that it takes no time to convert to html (at least for my archive), I think we can add a call to this at the end of parser.py:
- ask the user if they want to additionally convert the md files to html
- run the script if so

We could even run it by default apart from the fact that it needs an extra package.

Reply:
Or to avoid breaking things for people that have just downloaded parser.py, maybe we need to move the code into there. That would mean having the import line inside a function which is non-standard but supported I believe.